tenpy/linalg/symmetries.py

```diff
@@ -728,22 +721,51 @@ def factor_where(self, descriptive_name: str) -> int:
                 return i
         raise ValueError(f'Name not found: {descriptive_name}')
 
-    def qdim(self, a: Sector) -> int:
+    def qdim(self, a: Sector) -> float:
         if self.is_abelian:
             return 1
 
         dims = []
         for i, f_i in enumerate(self.factors):
             a_i = a[self.sector_slices[i]:self.sector_slices[i + 1]]
             dims.append(f_i.qdim(a_i))
-        return np.prod(dims)
+        #print('symmetry',type(np.prod(dims)))
+        return math.prod(dims)
+
+    def _f_symbol(self, a: Sector, b: Sector, c: Sector, d: Sector, e: Sector, f: Sector ) -> np.ndarray:
+        #print(self.factors)
+
+        contributions = []
+        for i, f_i in enumerate(self.factors):      #self =Fibonacci_L x Fibonacci_R Product symmetry i=index f_i= FibonacciGrading left, FibonacciGrading right
+
+            a_k = a[self.sector_slices[i]:self.sector_slices[i + 1]]
+            b_k = b[self.sector_slices[i]:self.sector_slices[i + 1]]
+            c_k = c[self.sector_slices[i]:self.sector_slices[i + 1]]
+            d_k = d[self.sector_slices[i]:self.sector_slices[i + 1]]
+            e_k = e[self.sector_slices[i]:self.sector_slices[i + 1]]
+            f_k = f[self.sector_slices[i]:self.sector_slices[i + 1]]
 
-    def _f_symbol(self, a: Sector, b: Sector, c: Sector, d: Sector, e: Sector, f: Sector
-                  ) -> np.ndarray:
-        raise NotImplementedError  # TODO
+
+            contributions.append(f_i._f_symbol(a_k, b_k, c_k, d_k, e_k, f_k)) #[μ,ν,κ,λ]
+
+
+        Fs=np.multiply(contributions[0],contributions[1])
```

Comment:
This looks incorrect if we have non-trivial multiplicity labels.
The total multiplicity label should go over all combinations over multiplicity labels.
So we need ``np.kron`` here, not ``np.multiply``, right?
We should clearly define how the multiplicity arises (C style or F style strides)